server/routes/locations.js

```diff
@@ -0,0 +1,44 @@
+const { isNilOrEmpty } = require('../utils/utils')
```

Comment:
I don't think it makes sense to group routes by the type of entity that they deal with. We tend to group routes around use cases. So in this case it would be better to call this `changePrison` rather than `locations`

Reply:
Done
makes sense